px4_ros2_cpp/include/px4_ros2/control/setpoint_types/experimental/fixedwing_lateral_longitudinal.hpp

```diff
@@ -71,5 +61,103 @@ class FwLateralLongitudinalSetpointType : public SetpointBase
     _longitudinal_control_limits_pub;
 };
 
+struct FwLateralLongitudinalSetpoint
+{
+  std::optional<float> course;
+  std::optional<float> airspeed_direction;
+  std::optional<float> lateral_acceleration;
+  std::optional<float> altitude_msl;
+  std::optional<float> height_rate;
+  std::optional<float> equivalent_airspeed;
```

Comment:
Are all of these optional? Which combinations are possible? What is required at the minimum?